crates/astria-conductor/src/executor/state.rs

```diff
@@ -379,7 +382,9 @@ mod tests {
 
     fn make_genesis_info() -> GenesisInfo {
         GenesisInfo::try_from_raw(raw::GenesisInfo {
-            rollup_id: vec![24; 32].into(),
+            rollup_id: Some(RawRollupId {
+                inner: vec![24; 32].into(),
+            }),
```

Comment:
Could do something like....

```suggestion
            rollup_id: Some(RollupId::new(vec![24; 32]).to_raw()),
```

In general throughout trying to avoid importing the raw types as much as possible, rely on the core types and their existing conversions.

I'd probably create the rollup id as it's own variable and then just call `.to_raw()` on it here instead of all nested to keep a bit cleaner